### ISSUE TITLE
NTR: Iodine low in urine (Lab: blood chemistry)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8428]
New term request Iodine low in urine (segal_190709130936): Iodide

**Comment by pnrobinson** [comment]
@MickeySegal

Adding new term: 
- ID: HP:6000438
- label: Reduced urinary iodine level
- parents: [Abnormal urinary mineral level (HP:0025640)](https://hpo.jax.org/app/browse/term/HP:0025640)
- pmid: 
- def: The amount of iodine in the urine, normalized for urine concentration, is above the upper limit of normal.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8428
- synonyms:

**Comment by grahamponting** [comment]
I appreciate this issue is closed and a new term has been created, however the definition is conflict with the label, that is one says "**reduced**" and the other says "**above the upper limit**", could you clarify please..
Graham Ponting

**Comment by MickeySegal** [comment]
The term name is correct.  

The definition needs to be fixed.

Thanks for noticing this and pointing it out.

**Comment by pnrobinson** [comment]
@grahamponting @MickeySegal --thanks for finding this error, I fixed it!
